db/migrate/20181221121805_create_theme_translation_override.rb

```diff
@@ -0,0 +1,11 @@
+class CreateThemeTranslationOverride < ActiveRecord::Migration[5.2]
+  def change
+    create_table :theme_translation_overrides do |t|
+      t.integer :theme_id, null: false
+      t.string :locale, length: 30, null: false
+      t.string :translation_key, null: false
+      t.string :value, null: false
+      t.timestamps null: false
+    end
```

Comment:
You will almost certainly want to create an index based on `theme_id` here at least.

It also seems to me that a unique index on `[:theme_id, :locale, :translation_key]` would be prudent.